tests/common/TestsSetup.hs

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+--
+-- Module      :  Main
+-- Copyright   :  (c) Oleg Grenrus 2015
```

Comment:
Actually, I just noticed this isn't new code, it's just been extracted verbatim. Since this code isn't written by me (or @garyb), I'm not at liberty to transfer copyright of it. If it's going to be relicensed back to the project under different copyright assumptions, it needs to be done under the terms of the original license, which means including that here. I think I'm going to put something in `CONTRIBUTING.md` about how copyright notices should be added, but I think at a minimum, the code must be new. IANAL though, etc. I hate to be pedantic about these things, but as I say, this isn't my code.